app/index.html

```diff
@@ -150,30 +150,38 @@
                     </li>
                 </ul>
                 <ul class="nav navbar-nav navbar-right">
+                    <li>
+                        <div>
+                            <form class="navbar-form" style="padding-right: 12px">
+                                <a class="btn btn-small btn-default" href="/automation-dashboard" target="/automation-dashboard"><img src="images/automation_dashboard_30.png" class="right-padding" style="height:25px">Automation Dashboard</a>
+                            </form>
+                        </div>
+                    </li>
+
+                    <li>
+                        <div>
+                            <form class="navbar-form" style="padding-right: 12px">
+                                <a class="btn btn-small btn-default active" href="/studio"><img src="images/studio_30.png" class="right-padding" style="height:25px">Workflow Studio</a>
+                            </form>
+                        </div>
+                    </li>
+
                     <li>
                     <div>
                     	<form class="navbar-form" style="padding-right: 12px">
-                    		<a class="btn btn-small btn-default" href="/scheduler" target="_blank"><img src="images/scheduler_30.png"class="right-padding" style="height:25px">Scheduling & Orchestration</a>
+                    		<a class="btn btn-small btn-default" href="/scheduler" target="/scheduler"><img src="images/scheduler_30.png"class="right-padding" style="height:25px">Scheduling & Orchestration</a>
```

Comment:
So we will no longer open a new window?

Reply:
This makes the link open in a new tab, and if the link has already been opened without closing it, it finds the right tab and re-opens it.